README.md

```diff
@@ -81,86 +82,43 @@ Requirements:
-
-#### Manually creating the configuration files
-
-Create a **test directory**, and include the following files:
-
-* `vars.json`:  The JSON file corresponding with the SIMP ISO.  Contains
-  variables that point to the ISO, the ISO checksum, the name of the
-  distribution ISO, and the `root` password. Samples can be found in
-  `samples/<sample>/vars.json`.
-* `simp_conf.yaml`:  YAML generated by `simp config`.  You can use a copy of
-  one from a previous build or edit one from from
-  `samples/<sample>/simp_conf.yaml`. Puppet server and FIPS configurations will
-  be overridden by `simp_config.rb` and `packer.yaml`.
-* `packer.yaml`:  Overrides defaults set in `simp_config.rb`.  Packer requires
-  this file (even if you don't want to override defaults). Samples can be found
-  in `samples/<sample>/packer.yaml`.  For CentOS 6 builds, you **MUST** change
-  the `nat_interface` and `host_only_interface` to `eth0` and `eth1`,
-  respectively.
-
-See [samples/README.md](samples/README.md) for more information.
+1. [Using `rake simp:packer:matrix`](#using-rake-simppackermatrix)
 
-#### Output file locations
+   - Builds multiple Vagrant boxes based on a matrix of settings.
+   - Publishes each box plus JSON files with **_versioned_** metadata to
+     a local directory tree.
+     - Beaker, `Vagrantfile`s, and `vagrant init` can use this JSON metadata to
+       request specific boxes and alert when a new version of a box is
+       available.
 
-- A **working directory** will be created under the test directory, where all the
-  scripts, manifests, and files are kept. It is erased on completion of the
-  test, but can be preserved with `SIMP_PACKER_save_WORKINGDIR=true`.
-- A time-stamped **log file** is created in the top level of the test directory and
-  all the log output from `packer` is copied there. If the `packer` build
-  fails, it is renamed from `<date>.log` to `<date>.log.errors`.
-- A **Vagrant box file** (`*.box`) with its `Vagrantfile` and `Vagrantfile.erb`
-  will be located in the directory defined by the `packer.yaml file`
-  `output_directory` (default is `<testdirectory>/OUTPUT`).
+2. [Using `rake simp:packer:build`](#using-rake-simppackerbuild)
 
+   - Builds a single Vagrant box, _without_ any version metadata.
+   - Requires you to [manually provide a test directory and configuration
+     files](#manually-creating-the-configuration-files)
 
+Both methods are configured by [environment variables](#environment-variables)
+and Rake task arguments.
 
-### Running a build matrix
+### Using `rake simp:packer:matrix`
 
 Often, it is desirable to build several 'flavors' of simp-packer boxes at the
 same time. A rake task, `simp:packer:matrix` has been provided for this purpose:
 
 ```sh
-[ENV_VAR=value ...] bundle exec rake simp:packer:matrix[MATRIX_ELEMENTS ...]
+TMPDIR=/some/tmp/dir \
+  [ENV_VAR=value ...] \
+  bundle exec rake simp:packer:matrix[MATRIX_ELEMENTS ...]
 ```
 
 A typical example would look like:
```

Comment:
The discussion about TMPDIR needs to be in this section as well.  See lines 180-181 in README.md in the PR.